packages/client/package.json

```diff
@@ -104,6 +105,9 @@
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11"
   },
+  "devDependenciesComments": {
+    "node-fetch": "Hotfix for client browser build error in older Node versions (12/14), #1305, 2021-06-18"
```

Comment:
oh nice idea to add some context like this, i may borrow this idea in the future :)

Reply:
Yeah, I just thought that this would be just prone to get lonst again over time and I googled a bit how to get around this annoying comment restriction for `json` files and that's what I found. 😄 

I would be relatively confident that this has no side effects, we might want to keep an eye on this though for some time (if there are for example resulting build problems or the like if one adds unknown keys to `package.json` files? 🤔 But again: likely not I would assume).